lib/engine/game/g_18_uruguay/farm.rb

```diff
@@ -79,7 +79,7 @@ def process_assign(action)
           target = action.target
           good = retreive_goods!(@farm_id)
 
-          target.assign!(good)
+          target.assign!(good, Engine::Assignable::STACKABLE)
```

Comment:
`target.assign!(good, stackable = true)` would better match how we've designed the interfaces in the framework.

Reply:
Changed to a generic stack_group: so that we can support multiple stack_groups

Reply:
Removed